ts/Core/Options.d.ts

```diff
@@ -73,6 +74,8 @@ export interface Options {
 }
 
 export interface DefaultOptions extends Options {
+    defaultCircularOptions?: RadialAxisOptions,
+    defaultRadialGaugeOptions?: RadialAxisOptions
```

Comment:
This should not be hardcoded her, instead in `RadialAxisOptions.d.ts` add a 
```ts
declare module '../Options' {
  // here goes the interface a second time with the properties for type merge
}
```

Reply:
Done!